.gitignore

```diff
@@ -9,4 +9,5 @@ docs/_build/
 docs/.vscode/*
 old/*
 .idea/
-bioverse.egg-info/
\ No newline at end of file
+bioverse.egg-info/
+testing/
```

Comment:
is this needed or can this be removed?

Reply:
I put my own testcases in that folder, so it's useful for me.